cockatrice/src/settingscache.cpp

```diff
@@ -89,6 +91,12 @@ void SettingsCache::setCardInfoViewMode(const int _viewMode) {
     settings->setValue("cards/cardinfoviewmode", cardInfoViewMode);
 }
 
+void SettingsCache::setHighlightWords(const QString &_highlightWords) {
+    // Words are seperated by a comma and you can not use spaces in words
+    highlightWords = _highlightWords.simplified().replace(" ", "").split(",");
```

Comment:
Would you like me to use the QString or QStringList plan then? I'm going to go with QString for now.